apps/chat-e2e/src/ui/webElements/attachFilesModal.ts

```diff
@@ -67,6 +113,25 @@ export class AttachFilesModal extends BaseElement {
     return this.allFilesTree;
   }
 
+  public getSectionElement(section: FileModalSection): Locator {
+    switch (section) {
+      case FileModalSection.AllFiles:
+        return this.rootLocator.locator(
```

Comment:
you may use `BaseElement` as a return type and transform the implementation like this:

`return this.getChildElementBySelector(AttachFilesModalSelectors.allFilesContainer,);`